torch_sim/models/orb.py

```diff
@@ -312,17 +312,15 @@ def __init__(
             self._device = torch.device(self._device)
 
         self._dtype = dtype
-        self._memory_scales_with = "n_atoms"
         self._compute_stress = compute_stress
         self._compute_forces = compute_forces
 
         # Set up system configuration
-        self.system_config = system_config or SystemConfig(
-            radius=6.0, max_num_neighbors=20
-        )
+        self.system_config = system_config or model.system_config
         self._max_num_neighbors = max_num_neighbors
         self._edge_method = edge_method
         self._half_supercell = half_supercell
+        self._conservative = conservative
```

Comment:
should any of these be exposed as public / can be changed on the fly?

Reply:
Pulled the `conservative` attribute out.